src/main/java/uk/ac/cam/cl/dtg/isaac/dos/IsaacQuiz.java

```diff
@@ -74,4 +78,8 @@ public boolean getVisibleToStudents() {
     public void setVisibleToStudents(boolean visibleToStudents) {
         this.visibleToStudents = visibleToStudents;
     }
+
+    public Content getRubric() {return rubric;}
+
+    public void setRubric(Content rubric){this.rubric = rubric;}
```

Comment:
Oh, this isn't pretty! Could it stick to our "one statement per line" code style? (I know IntelliJ makes it look like this for you, but the code itself is 3 lines elsewhere!)

Reply:
Yeah, oops, as you say IntelliJ made it look correct and I didn't check line numberings!